docs/components/stats.md

```diff
@@ -8,27 +8,33 @@ source_code: src/components/scene/stats.js
 
 [scene]: ../core/scene.md
 
-The stats component displays a UI with performance-related metrics. The stats
-component applies only to the [`<a-scene>` element][scene].
+The stats component displays a UI with performance-related metrics of your Aframe Project. The stats component applies only to the [`<a-scene>`][scene] element.
 
 ## Example
 
 ```html
 <a-scene stats></a-scene>
 ```
+It can also be written as `stats="true"` which will activate the UI and `stats="false"` to hide the UI. This version can be useful during development as it allows you to keep the component within your code for easy access. 
```

Comment:
This should be documented in a values section like this one https://aframe.io/docs/1.4.0/components/gltf-model.html#values